src/progress/mod.rs

```diff
@@ -179,6 +185,11 @@ impl Progress {
         need_update
     }
 
+    /// update committed_index.
+    pub fn update_committed(&mut self, committed_index: u64) {
+        self.committed_index = committed_index
```

Comment:
Should skip smaller value.

Reply:
I think the committed index is non-decreasing, so we wont get the smaller value.
What I miss here?

Reply:
Message can be reordered.